keras_core/ops/symbolic_arguments.py

```diff
@@ -5,7 +5,21 @@
 
 class SymbolicArguments:
     def __init__(self, *args, **kwargs):
-        # TODO: validation
```

Comment:
I think you can just remove the TODO. The checks below aren't really useful since they would never get triggered in practice (and they add some overhead).

Reply:
Hi @fchollet ,

I have removed the validation checks and TODO statement, kindly review them,